packages/kubiscan/PKGBUILD

```diff
@@ -0,0 +1,47 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=kubiscan
+_pkgname=KubiScan
+pkgver=195.b851b37
+pkgrel=1
+pkgdesc='A tool for scanning a Kubernetes cluster for risky rbac permissions.'
+arch=('any')
+groups=('blackarch' 'blackarch-scanner')
+url='https://github.com/cyberark/KubiScan'
+license=('GPL3')
+depends=('python' 'python-virtualenv')
+makedepends=('git' 'python-pip')
+source=("git+https://github.com/cyberark/$_pkgname.git")
+sha512sums=('SKIP')
+install="$pkgname.install"
+
+pkgver() {
+  cd $_pkgname
+
+  echo "$(git rev-list --count HEAD).$(git rev-parse --short HEAD)"
+
+}
+
+package() {
+  cd $_pkgname
+
+  install -dm 755 "$pkgdir/usr/bin"
+  install -Dm 644 requirements.txt "$pkgdir/usr/share/$pkgname/requirements.txt"
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" *.md
+
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+
+  rm -rf LICENSE *.md
+
+  cp -a * "$pkgdir/usr/share/$pkgname/"
+
+  cat > "$pkgdir/usr/bin/$pkgname" << EOF
+#!/bin/sh
+source /usr/share/$pkgname/venv/bin/activate
```

Comment:
Is that really justified to use a virtual env here? there are only 2 deps: https://github.com/cyberark/KubiScan/blob/master/requirements.txt
`python-kubernetes` is already in AL repos.
And https://github.com/kxxoling/PTable seems to be pure python withotu any external dep so very easy to package and maintain.
Sounds like a virtual env is not required here.
Could you repackage with the classic setuptools setup?

Reply:
Let me look. I did not see a setup.py in the repo ?

Reply:
There is no setup.py and fails. Fixed tags; resolved. Let me know how you would like to proceed. 

Thanks.

Reply:
> Let me look. I did not see a setup.py in the repo ?

a classic jointscript then.

Reply:
^^ I do not know what you mean by that ?

Reply:
By jointscript I mean something similar to https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-generic-jointscript

Because there is no semi-auto packaging method available, you have to deploy the python script, a wrapper, documentation license, etc. manually.

Reply:
I see. I was thinking something else. i will update. Thanks.

Reply:
You may find exampels on closed issues with `setyp.py::false` label https://github.com/BlackArch/blackarch/issues?q=is%3Aissue+label%3Asetup.py%3A%3Afalse+is%3Aclosed that have an associated PR.

Reply:
Yeah this is not going to work. May scrap this. All kinds of errors.

Reply:
Going open a request for a setup or pyproject.toml.

Reply:
Noting: cyberark will create a setup once they can get too it. Please hold. 
https://github.com/cyberark/KubiScan/issues/68

Thanks.